### ISSUE TITLE
filter not available in Gimp 2.10.18

### ISSUE THREAD

**Issue by lukruh** [DASPRiD/gimp-telegram-sticker#2]
Hey, I have used the script in previous versions of gimp already, but now the filter does not appear in the filter menu anymore. I double checked the folders and tryed system and user plug-ins and scripts folders (ubuntu 20.04) any ideas, what I am missing?

**Comment by DASPRiD** [comment]
Not sure, I just tested with Gimp 2.10.28 on Pop_OS 20.04 (Flatpak version), and it's working fine. Please double check:

Open Preferences > Folders > Plugins and check that you are using the correct plugin folder. The folder does change with each version.

**Comment by lukruh** [comment]
Thanks for the fast response.
I double checked the folder already and just did so another time. When I open one of the Plugins folders through Gimp's Preferences I can see that `telegram-sticker.py` is in that folder. 

Update: I tryed to run Gimp from commandline to see errors and warnings, 
`/usr/bin/env: ‘python’: No such file or directory`, seems to be related. I fixed that by linking `python` to `python3`. Now I can see Gimp trying to load the plugin, but failing in ~/.config/GIMP/2.10/plug-ins/telegram-sticker.py: 
```
    from gimpfu import *
ModuleNotFoundError: No module named 'gimpfu'
```
I searched the web and found that gimp packages in some distributions are compiled without  some python related dependencies, looks like this is the case for my version, going to try the flatpack version before recompiling it manually.

Update2: installing gimp using flatpak works fine, pygimp seems to be included. Thanks I can create tons of stickers again :)
